build-executor/pom.xml

```diff
@@ -129,6 +129,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+      <dependency>
+          <groupId>com.openshift</groupId>
+          <artifactId>openshift-restclient-java</artifactId>
+      </dependency>
```

Comment:
Build executor should not depend on Openshift client. The Openshift client is an implementation detail of the driver.